src/Symfony/Component/ErrorHandler/Error/FatalErrorInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\Error;
+
+interface FatalErrorInterface extends \Throwable
+{
+    /**
+     * @return array An array as returned by error_get_last()
```

Comment:
No need for the `@return` duplicating the return type.

```suggestion
     * Returns an array as returned by error_get_last()
```